modules/policy/engine/src/test/java/org/eclipse/sw360/antenna/policy/engine/TestUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+public class TestUtils {
+    /**
+     * Helper for getting the element at index x in a collection
+     *
+     * @param data The collection from which the element should be retrieved
+     * @param index The index of the element to be retrieved
+     * @param <T> The data type used in the collection
+     * @return The element found
+     * @throws IndexOutOfBoundsException If the index is bigger than the collections size
+     */
+    public static <T> T getIndex(final Collection<T> data, final int index) {
```

Comment:
Maybe use [CollectionUtils](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html#get-java.lang.Object-int-) instead? (Assuming commons-collections is already in use; it probably does not pay-off to introduce a new dependency just for that helper.)

Reply:
That was the trade of, I know of all those cool Apache utils, but they are all dependencies :-).

Reply:
see also: https://github.com/eclipse/antenna/pull/260#discussion_r316152345 and https://github.com/eclipse/antenna/pull/260#discussion_r316148041

Reply:
Added the CollectionUtils dependency with scope test

Reply:
why not instead just make it a `List`? A collection where you want to be able to get the nth element is by definition a List, and if the collection would be a Set, this function has undeterministic behaviour?